aibolit/utils/utils.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+
+class RemoveComments:
```

Comment:
@lyriccoder Here too. Remove this guy from `utils.py` and create a `py` file for this class only. Let's avoid utility files / libraries. Also, let's think in this class declaratively, not imperatively: this should be a decorator which would receive a string in its constructor and return the string without comments in its `value` method.